### ISSUE TITLE
🐛 [ddns-updater] No access to the web-interface

### ISSUE THREAD

**Issue by Wiz0r** [runtipi/runtipi-appstore#4586]
### Store Application

ddns-updater

### App version

2.7.0

### Description

The access to the administration page works neither via HTTP nor HTTPS.

The HTTPS version states a 404 Error. The HTTP Version is not accessible at all.

### Steps to reproduce

1. Install ddns-updater
2. Try to access the admin page

### App logs

```bash
ddns-updater  | Running version 2.7.0 built on 2024-06-26T19:34:57.277Z (commit d37f057)
ddns-updater  |
ddns-updater  | 🔧 Need help? https://github.com/qdm12/ddns-updater/discussions/new
ddns-updater  | 🐛 Bug? https://github.com/qdm12/ddns-updater/issues/new
ddns-updater  | ✨ New feature? https://github.com/qdm12/ddns-updater/issues/new
ddns-updater  | ☕ Discussion? https://github.com/qdm12/ddns-updater/discussions/new
ddns-updater  | 💻 Email? quentin.mcgaw@gmail.com
ddns-updater  | 💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
ddns-updater  | 2024-08-27T08:23:09Z INFO Settings summary:
ddns-updater  | ├── HTTP client
ddns-updater  | |   └── Timeout: 10s
ddns-updater  | ├── Update
ddns-updater  | |   ├── Period: 5m0s
ddns-updater  | |   └── Cooldown: 5m0s
ddns-updater  | ├── Public IP fetching
ddns-updater  | |   ├── HTTP enabled: yes
ddns-updater  | |   ├── HTTP IP providers
ddns-updater  | |   |   └── all
ddns-updater  | |   ├── HTTP IPv4 providers
ddns-updater  | |   |   └── all
ddns-updater  | |   ├── HTTP IPv6 providers
ddns-updater  | |   |   └── all
ddns-updater  | |   ├── DNS enabled: yes
ddns-updater  | |   ├── DNS timeout: 3s
ddns-updater  | |   └── DNS over TLS providers
ddns-updater  | |       └── all
ddns-updater  | ├── Resolver: use Go default resolver
ddns-updater  | ├── Server
ddns-updater  | |   ├── Listening address: :8000
ddns-updater  | |   └── Root URL: /
ddns-updater  | ├── Health
ddns-updater  | |   └── Server listening address: 127.0.0.1:9999
ddns-updater  | ├── Paths
ddns-updater  | |   ├── Data directory: /updater/data
ddns-updater  | |   └── Config file: /updater/data/config.json
ddns-updater  | ├── Backup: disabled
ddns-updater  | └── Logger
ddns-updater  |     ├── Level: info
ddns-updater  |     └── Caller: hidden
ddns-updater  | 2024-08-27T08:23:09Z INFO reading JSON config from file /updater/data/config.json
ddns-updater  | 2024-08-27T08:23:09Z INFO file not found, creating an empty settings file
ddns-updater  | 2024-08-27T08:23:09Z ERROR cannot write configuration to file: open /updater/data/config.json: permission denied
ddns-updater  | 2024-08-27T08:23:09Z INFO Shutdown successful
```

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of RunTipi, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by steveiliop56** [comment]
Hello @Wiz0r,

This is indeed an issue on our end and you we will release an update today fixing it.

**Comment by steveiliop56** [comment]
Fixed in #4587

**Comment by Wiz0r** [comment]
Looks good, thank you very much.
